index.html

```diff
@@ -288,10 +288,10 @@ <h1>Sponsors</h1>
             <a aria-label="HackNC snapchat" target="_blank" href="https://www.snapchat.com/add/hacknc">
                 <i class="fa fa-snapchat-square fa-2x"></i></a>
             <br>
-            <u>© HackNC 2021 — </u>
+            <u>© HackNC 2023 — </u>
```

Comment:
Can you remove the dash at the end? not sure why it was there before

Reply:
alright, on it!